src/types/react-table-config.ts

```diff
@@ -89,7 +89,9 @@ declare module 'react-table' {
       UsePaginationInstanceProps<D>,
       UseRowSelectInstanceProps<D>,
       UseRowStateInstanceProps<D>,
-      UseSortByInstanceProps<D> {}
+      UseSortByInstanceProps<D> {
+    initialRows: Row<D>[];
+  }
```

Comment:
Formatting messed up here?

Reply:
Everything good here.

Reply:
![image](https://user-images.githubusercontent.com/9084735/127497904-5dbf6d6d-99b1-4173-b1e9-a02b7ce3c7fb.png)

Reply:
It is a property, not an extended interface/type.
![image](https://user-images.githubusercontent.com/36186912/127498095-54b74ea0-5f8a-4fa4-9939-47e11c65290e.png)

Reply:
Ohhh the braces are for the main interface, that's confusing lol